### ISSUE TITLE
Missing dict file if "Pretrained model" checked

### ISSUE THREAD

**Issue by sot-tech** [wladradchenko/wunjo.wladradchenko.ru#15]
If UI "Pretrained model" checked, operation fails with empty descriptive message (just `Error training...`).
Traceback:
```
AssertionError
	assert os.path.exists(dict_source)
File "/usr/lib/wunjo/app/speech/tps/tps/utils/__init__.py", line 44, in load_dict
	self.entries = load_dict(dict_source, fmt)
File "/usr/lib/wunjo/app/speech/tps/tps/modules/custom/replacer.py", line 30, in __init__
	super().__init__(dict_source, "Omograph")
File "/usr/lib/wunjo/app/speech/tps/tps/modules/omographs/rule_based/independent.py", line 14, in __init__
	super().__init__(dict_source, prefer_user)
File "/usr/lib/wunjo/app/speech/tps/tps/modules/omographs/rule_based/russian.py", line 10, in __init__
	md.RuOmograph([omographs_dict, "plane"], True),
File "/usr/lib/wunjo/app/speech/tps/tps/handler.py", line 487, in _get_default_modules
	modules = _get_default_modules(charset, data_dir, verify_checksum, silent, use_cleaner)
File "/usr/lib/wunjo/app/speech/tps/tps/handler.py", line 384, in from_charset
	text_handler = Handler.from_charset(hparams.charset, data_dir=os.path.join(os.path.dirname(os.path.abspath(__file__)), "data"), silent=True)
File "/usr/lib/wunjo/app/tacotron2/train.py", line 104, in prepare_dataloaders
	train_loader, valset, collate_fn = prepare_dataloaders(hparams, distributed_run)
File "/usr/lib/wunjo/app/tacotron2/train.py", line 278, in train
	train(hparams, distributed_run=distributed_run, rank=rank, n_gpus=n_gpus)
File "/usr/lib/wunjo/app/tacotron2/train.py", line 412, in main
	tacotron2(hparams_path=config_path)
```
`dict_source` in that case contains path `/usr/lib/wunjo/app/tacotron2/data/omographs.dict` (missing in distribution).

Tech info:
* OS: ubuntu 22.04.3 x64
* Python version: 3.10.12
* wunjo version: 1.5.3
* Installation source: https://wladradchenko.ru/static/wunjo.wladradchenko.ru/build/linux/wunjo_1.5.3.deb

**Comment by sot-tech** [comment]
Upd: manual copy of files `omographs.dict`, `stress.[a,b,c].dict` from `/usr/lib/wunjo/app/speech/tps/rules/` into `/usr/lib/wunjo/app/tacotron2/data/` fails with next error:
```
KeyError: 'state_dict'
	pretrained_dict = checkpoint_dict["state_dict"]
File "/usr/lib/wunjo/app/tacotron2/train.py", line 144, in warm_start_model
	model = warm_start_model(
File "/usr/lib/wunjo/app/tacotron2/train.py", line 285, in train
	train(hparams, distributed_run=distributed_run, rank=rank, n_gpus=n_gpus)
File "/usr/lib/wunjo/app/tacotron2/train.py", line 412, in main
	tacotron2(hparams_path=config_path)
File "/usr/lib/wunjo/app/train/tacotron2train.py", line 37, in run_train
	Tacotron2Train.run_train(config_path)
File "/usr/lib/wunjo/app/train/tacotron2train.py", line 27, in train
	Tacotron2Train.train(param)
File "/usr/lib/wunjo/app/train/utils.py", line 19, in training_route
	training_route(param)
File "/usr/lib/wunjo/app/wunjo/app.py", line 714, in common_training_route
	Traceback (most recent call last):
```

**Comment by wladradchenko** [comment]
Interest. I will solve this path bug in version 1.5.4. Now, you can copy those files as you did. Your error `checkpoint_dict["state_dict"]` means what in checkpoint witch you use was trained on another version of Tacotron and doesn't have key "state_dict". It means what you need to train the tacotron2 model by Wunjo AI and after you can continue train the tacotron2 model as "Pretrained model".

**Comment by wladradchenko** [comment]
Solved in version 1.5.4
